capa/render/result_document.py

```diff
@@ -501,9 +510,23 @@ class MaecMetadata(FrozenModel):
     model_config = ConfigDict(frozen=True, populate_by_name=True)
 
 
+@lru_cache(maxsize=None)
+def load_rules_prevalence() -> Dict[str, str]:
+    CD = Path(__file__).resolve().parent.parent.parent
+    file = CD / "assets" / "rules_prevalence_data" / "rules_prevalence.json.gz"
```

Comment:
use `get_default_root()` https://github.com/mandiant/capa/blob/210a13d94ea40114e01b40927c77c1c74047780f/capa/main.py#L444

Reply:
using `get_default_root` works well locally but it cause circular import when being during pyinstaller build. I suggest moving such functions to `capa.helpers`.

Reply:
moving it makes sense (see #1821 also)

Reply:
@mr-tz I suggest we move ahead with proposal 3 in above mentioned PR.
moving below to a new `capa.loader` or we can move them to `capa.helper`
```
has_file_limitation
is_supported_format
is_supported_arch
get_arch
is_supported_os
get_os
is_running_standalone
get_default_root
get_default_signatures
get_workspace
get_extractor
get_file_extractors
get_signatures
get_sample_analysis
collect_metadata
compute_dynamic_layout
compute_static_layout
compute_layout
```

Reply:
sounds good to me!